src/main/java/uk/gov/hmcts/ccd/data/casedetails/search/SearchQueryFactoryOperation.java

```diff
@@ -54,8 +55,9 @@ public SearchQueryFactoryOperation(CriterionFactory criterionFactory,
     public Query build(MetaData metadata, Map<String, String> params, boolean isCountQuery) {
         final List<Criterion> criteria = criterionFactory.build(metadata, params);
 
+        Map<String, Object> queryParams = Maps.newHashMap();
```

Comment:
minor. If you want can you rename `queryParams` to `boundParameters`? we use `queryParams` in other parts of the code to refer to the query params coming in to the query endpoint.